packages/scene-composer/src/components/panels/ComponentEditMenu.tsx

```diff
@@ -21,6 +21,8 @@ interface ComponentEditMenuProps {
 enum ObjectTypes {
   EditComponent = 'edit-component',
   AddDataBinding = 'add-data-binding',
+  AddEntityBinding = 'add-entity-binding',
```

Comment:
is it used anywhere?

Reply:
Yes. This is used in the addComponentMenu.